.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,37 @@
+name: "\U0001F41B Bug Report"
+description: Report a bug on huggingface_hub
+labels: [ "bug" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If you intend to submit a PR for this issue, tell us in the description. Thanks!
+      placeholder: Bug description
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction
+      description: Please provide a code sample that reproduces the problem you ran into. It can be a Colab link or just a code snippet.
+      placeholder: Reproduction
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: "Please include the Python logs if you can."
+      render: shell
+  - type: textarea
+    id: system-info
+    attributes:
+      label: System Info
+      description: Please share your system info with us,
+      render: shell
+      placeholder: huggingface_hub version, Python Version, etc
```

Comment:
For this it would be super nice to have a `huggingface-cli env` command like we do for other libraries.

If you feel like doing it, we follow roughly the same structure as `transformers` in terms of commands, and it is visible [here](https://github.com/huggingface/transformers/blob/main/src/transformers/commands/env.py), registered [here](https://github.com/huggingface/transformers/blob/main/src/transformers/commands/transformers_cli.py#L36).

Not necessary for this PR to be merged though.